ci/dev-pipeline.yml

```diff
@@ -203,7 +206,7 @@ jobs:
             params:
               BUILD_TYPE: 0
               GH_TOKEN: ((github_access_token))
-              SIGNING_KEY: ((github_ons_spp_machine_user_gpg_key_private))
+              SIGNING_KEY: ((github_ons_spp_machine_user_gpg_key_private))]
```

Comment:
Yes it will fail at the moment as the pipline is set via fly to run another pipeline. Once the PR is approved I will be running it again on the original (There is conflict between mine and Joes changes at the moment). Thanks for the spot i will correct it.